source/_integrations/emoncms.markdown

```
@@ -28,6 +28,15 @@ To write information from Home Assistant to Emoncms, you can use the [`emoncms_h
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+URL:
+  description: The url of the emoncms server starting with the protocol (http or https). Example : https://emoncms.org
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)